### ISSUE TITLE
Decide on title for "Candidate and committee services" section

### ISSUE THREAD

**Issue by emileighoutlaw** [18F/fec-cms#855]
We never made this into an issue, but we need to make a decision before we can launch. Currently being considered:

> Candidate and committee resources
> Candidate and committee information
> Help for candidates and committees

cc @AmyKort

**Comment by emileighoutlaw** [comment]
@nickykrause — I was trying to keep an eye on usability findings, but I fear I missed any new data that came up about the name of this section. 

Any chance you can point me in the right direction, if we posted an issue in the usability findings repo?

**Comment by emileighoutlaw** [comment]
Hey @AmyKort — I chatted with Nicky about the findings from usability research. There's a long answer and a short answer.

**The short answer** is that in general folks found `Help for candidates and committees` to be the most inviting name. So I think we should give that a go for a title, if that sounds good to content managers.

**The longer answer** is that:

`Candidate and committee support` was confusing to some folks (who thought it meant to support, as in contribute to a group). 

In general, the options with `information` and `resources` felt a little too vague to some people (i.e., they weren't totally sure what they'd be getting).

There's nothing that should prevent us from moving forward with `Help for candidates and committees`, but I do think there are a couple things to keep an eye on moving forward:
- `Help` might be a little misleading as to how much human assistance is available when users click into the section. The section is majority informational and not hands-on assistance (although we are working to build pathways to contact information). So I'd like to keep an eye on that. 
- `Help` could have been subconsciously appealing to the users we tested with, most of whom were familiar with the old website (including _Help with reporting and compliance_). So their preference could have been from that subconscious recognition. I don't think there's really a way to quantify that, but we can see how intuitive the section is for new users in future testing.
